lib/npg_pipeline/function/pp_archiver.pm

```diff
@@ -11,9 +11,8 @@ use Try::Tiny;
 use Perl6::Slurp;
 
 use npg_qc::Schema;
-use npg_pipeline::function::definition;
 
```

Comment:
TODO - to be converted! NB - two functions map to this class

Reply:
D'oh!